sbi/neural_nets/flow.py

```diff
@@ -417,8 +422,100 @@ def mask_in_layer(i):
     # Combine transforms.
     transform = transforms.CompositeTransform(transform_list)
     neural_net = flows.Flow(transform, distribution, embedding_net)
+    flow = NFlowsFlow(neural_net, condition_shape=batch_y[0].shape)
 
-    return neural_net
+    return flow
+
+
+def build_zuko_maf(
```

Comment:
Great!

Reply:
I see that there is a test below, but it seems Codecov does not recognize it or a lot of lines are not covered by the test?